css/main.css

```diff
@@ -1,414 +1,658 @@
-           border-radius: 5px;
-           align-self: flex-start;}
-
-.drift {width: 100%;}
-
-.blog-wrapper {display: flex;
-               flex-flow: column nowrap;
-               align-items: center;
-               padding: 60px 40px;
-               line-height: 28px;
-               background-color: #ececec;}
-
-.blog__content__h {font-size: 32px; line-height: 36px;}
-
-.blog-p__initial-p {align-self: flex-start;
-                    color: #9e9ea7;}
-
-.read-link {color: #464646;
-            text-decoration: none;
-            font-size: 14px;
-            margin-top: 20px;
-            align-self: flex-start;}
-
-.connect-wrapper {background-color: #488283;
-                  color: white;
-                  text-align: center;
-                  line-height: 34px;
-                  padding: 40px 40px 20px;}
-
-.connect-wrapper__p {margin-bottom: 30px;}
-
-.bg1, .bg2, .bg3, .bg4 {background-repeat: no-repeat;
-                        background-size: cover;
-                        line-height: 300px;
-                        min-height: 300px;
-                        margin-top: 3px;}
-
-.bg1 {background-image: url(../img/docs-bg.png);}
-
-.bg2 {background-image: url(../img/samples-bg.png);}
-
-.bg3 {background-image: url(../img/slack-bg.png);}
-
-.bg4 {background-image: url(../img/github-bg.png);}
-
-.sdm, .code, .slack, .github {width: 40px;}
-
-.bg-p {color: white;
-       text-align: center;
-       font-size: 21px;}
-
-.bg1-img,
-.bg2-img,
-.bg3-img,
-.bg4-img {display: flex;
-          flex-flow: row nowrap;
-          vertical-align: middle;
-          justify-content: center;
-          position: relative;
-          top: 120px;
-          align-items: center;}
-
-/* Testimonials */
-
-.marlette,
-.kyyti,
-.nvidia,
-.pivotal,
-.barclays {width: 45px;}
-
-.slide1-h,
-.slide2-h,
-.slide3-h,
-.slide4-h,
-.slide5-h {float: left;
-           color: #717274;}
-
-.slide-h {font-size: 16px;}
-
-.slide1-img-wrapper,
-.slide2-img-wrapper,
-.slide3-img-wrapper,
-.slide4-img-wrapper,
-.slide5-img-wrapper {float: right;}
-
-.slide1,
-.slide2,
-.slide3,
-.slide4,
-.slide5 {clear: both;
-         padding: 0px 40px 40px 40px;
-         margin-top: 55px;
-         height: 100%;
-         background-color: #f7f7f7;}
-
-.slide1 p,
-.slide2 p,
-.slide3 p,
-.slide4 p,
-.slide5 p {margin-bottom: 40px; font-size: 18px; line-height: 1.5em;}
-
-.quotes {position: relative; top: -20px;}
-
-/* Slideshow container */
-.slideshow-container {
-  max-width: 100%;
-  position: relative;
-  margin: auto;
+* {
+  margin: 0;
+  padding: 0;
 }
 
-/* Hide the images by default */
-.mySlides {
-  display: none;
+body {
+  font-family: "Segoe UI", Tahoma, Geneva, Verdana, sans-serif;
 }
 
+/* Mobile */
 
-/* The dots/bullets/indicators */
-.dot {
-  cursor: pointer;
-  height: 5px;
-  width: 5px;
-  margin: 0 5px;
-  background-color: #bbb;
-  border-radius: 50%;
-  display: inline-block;
+@media screen and (min-width: 320px) and (max-width: 1024px) {
```

Comment:
This (max-width: 1024px) overlaps with the tablet version, we should reduce it to (max-width: 768px)